packages/js/data/src/import/selectors.ts

```diff
@@ -6,12 +6,12 @@ import { ImportState, ImportStatusQuery, ImportTotalsQuery } from './types';
 
 export const getImportStarted = ( state: ImportState ) => {
 	const { activeImport, lastImportStartTimestamp } = state;
-	return { activeImport, lastImportStartTimestamp } || {};
```

Comment:
typescript picked this up as an expression that always evaluates to the left hand side because even if `activeImport` and `lastImportStartTimestamp` are null, the left hand side will still evaluate to `{}`